src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -1709,11 +1712,33 @@ private void HandleException(_EXCEPTION_POINTERS* pExceptionPointers)
             throw new NotSupportedException("HandleException");
         }
 
+        private bool runWithErrorTrap(void* function, void* parameter)
+        {
+            // This method is completely handled by the C++ wrapper to the JIT-EE interface,
+            // and should never reach the managed implementation.
+            Debug.Assert(false, "CorInfoImpl.runWithErrorTrap should not be called");
+            throw new NotSupportedException("runWithErrorTrap");
+        }
+
         private void ThrowExceptionForJitResult(HRESULT result)
         { throw new NotImplementedException("ThrowExceptionForJitResult"); }
         private void ThrowExceptionForHelper(ref CORINFO_HELPER_DESC throwHelper)
         { throw new NotImplementedException("ThrowExceptionForHelper"); }
 
+        private uint SizeOfPInvokeTransitionFrame
+        {
+            get
+            {
+                // struct PInvokeTransitionFrame:
+                //  m_RIP
+                //  m_FramePointer
+                //  m_pThread
+                //  m_dwFlags + align
+                //  m_PreserverRegs - RSP
```

Comment:
`m_PreservedRegs` is a variable-sized array -- are you doing something special in the JIT to line the frame up with the normal register spills?  Should there be some additional comments in here about how this works?